src/App.js

```diff
@@ -1,8 +1,34 @@
 import React from 'react';
-import Calculator from './components/Calculator';
-
-const App = () => (
-  <Calculator />
-);
+import { Link, Route, Routes } from 'react-router-dom';
+import HomePage from './pages/HomePage';
+import CalculatorPage from './pages/CalculatorPage';
+import QuotePage from './pages/QuotePage';
+import Footer from './Footer';
 
+function App() {
```

Comment:
- [ ] Good job @batoorsohail, You'r app is almost complete. The only thing remaining is to capitalize the name for your footer.js component. As you can see you create your file with a small `f`/`footer.js` but your import is with a capital `F`/`Footer` which is causing a problem to start your app. Please rename your `footer.js` to `Footer.js`
![image](https://user-images.githubusercontent.com/69077061/223497741-2af76196-016c-46b4-8d7a-68ee3ba93530.png)

- [Optional] I would also suggest adding your footer file inside your `components` folder, as we can agree `footer` is another component of the page.